board/piksiv3/linux.config

```diff
@@ -208,3 +207,9 @@ CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_RCU_CPU_STALL_TIMEOUT=60
 # CONFIG_FTRACE is not set
+CONFIG_RPMSG=m
```

Comment:
Why not put these in the kernel proper?
Before, remoteproc had to be modular to allow restarting the slave processor, but this is now allowed for in the sysfs interface.
It also seems perfectly reasonable to put the FPGA drivers in the kernel proper.

Reply:
Agree, if these don't need to be modules, let's compile them into the kernel.

Reply:
done